config/config.go

```diff
@@ -255,8 +255,8 @@ type ProxyConfig struct {
 	// ConsumerAuthorization defines the proxy-level consumer authorization
 	ConsumerAuthorization ConsumerAuthorization `yaml:"consumer_authorization,omitempty" json:"consumer_authorization,omitempty"`
 
-	// Name of the target server for this proxy.
-	Target string `yaml:"target" json:"target"`
+	// Transformation rules applied to HTTP request headers.
+	RequestHeaderTransforms RequestHeaderTransformations `yaml:"request_header_transforms,omitempty" json:"request_header_transforms,omitempty"`
```

Comment:
I like this change a lot. A couple questions: 1. Should this be (or also be) in Operation? 2. The next thing would be a RequestBodyTransformation, so I'm wondering if we have a "RequestTransformations" section or if we leave them top-level?

Reply:
Is ext_authz able to tamper with request body?

Reply:
No ext_authz cannot change the request body, but ext_proc can. I don't have a preferences between having multiple top-level transform sections versus grouping them under a single `RequestTransforms` section. In general I prefer whatever keeps the config the flattest, and I think both are 2 levels deep.

Reply:
And yes, it is already in the Operation (see line 281).

Reply:
Ok. Grouping or not is fine with since I assume there will only ever be two anyway: Headers and Body.

Reply:
We probably also want to do first-class transforms for special values like URL and HTTP method. That makes me think we _should_ add `HTTPRequestTransforms` as a grouping, with everything within. Lemme update.

Reply:
PTAL

Reply:
Looks good. So we're supporting all these transforms for next release?

Reply:
If time allows!